src/routes/formsgSiteCreation.ts

```diff
@@ -82,20 +81,20 @@ export class FormsgRouter {
       if (!siteName) {
         const err = `A site name is required`
         await this.sendCreateError(requesterEmail, repoName, submissionId, err)
-        return res.sendStatus(200)
+        return cloneRes.sendStatus(200)
```

Comment:
where is cloneRes coming from?

Reply:
Where is the clone logic?

Reply:
The rename in vscode renamed all instances instead of the variable...

I didn't catch this cos I didn't expect it to do a global rename of the word instead of the variable...

Will update?